add section how to transfer your files simple to your server

tzabbi opened the pull request:
Add a guide which explains how to create a directory which will be used by the server. With this you can simply add custom files to your server.

tzabbi committed:
add section how to transfer your files simple to your server

jusito commented:
Heyho, I don't know if advising people to use bind mount is a good idea.
1. The current image can't handle permission issues well. (Sure, fixable)
2. Docker bind mount should be turned off for security reasons (any user mount "/" even malicious script). So there are configurations where bind mount will not work., therefore this readme wouldn't work.
3. On some systems like WSL bind mount can come with a heavy performance penalty, e.g. if you bind mount a Windows to Linux shared folder.

But because it's for some reason quite often used and impacts the user I agree that there should be a note. Maybe in entrypoint checking if the permissions are fine and if not throwing a good error message with "What the issue? Whats the cause? How to solve?" e.g. "File permissions in /home/steam preventing correct Container execution. Some files in /home/steam aren't owned by the correct user steam with uid 10000. Please fix with `docker run -it --rm -v VolumeName:/home alpine chown -R 10000 /home .`

jusito commented:
In essence there is my question: why do you prefer bind mount instead of volumes?

tzabbi commented:
You are totally right. For me it worked but I only uses Linux Server and Clients.
1. What do you mean?
2. I don't know that the bind mount is an security issue in combination with docker. So we should avoid it.
3. I don't use WSL so I can't say something about this point.

I'll add this section and remove my points.

I just want add this bind mount, because it is easy to copy files to the container.

Commit:
remove bind mount and add permission issue fix:
- bind mount and docker has security issues with docker and performance issues with WSL
- not every file has owner and group id 10000
